### DISCUSSION TITLE
Videojs and ssa/ass subtitles

### DISCUSSION THREAD

**Discussion by Oniara** [videojs/video.js#8667] [Q&A] [answered]
Hello there! I am new to Github so sorry if I am doing something wrong!

Anyhow, I have a question if there is any way I could use ssa/ass subtitles in videojs? Right now I use .vtt subtitles by adding them through remote text track. But I really need the capabilities of ssa/ass subtitles.

**Comment by amtins** [comment]
@Oniara there is no native support for video.js, but there is the [videojs-ass](https://www.npmjs.com/package/videojs-ass) plugin. Have you had a chance to try it out?

Repo source code : https://github.com/SunnyLi/videojs-ass

**Reply by Oniara** [reply]
Hey there! I am pretty sure this plugin is outdated sadly. Didn't mange to get it work.

**Reply by mister-ben** [reply, marked as answer]
You might try https://github.com/libass/JavascriptSubtitlesOctopus. The example in their readme uses video.js:

https://libass.github.io/JavascriptSubtitlesOctopus/videojs.html.

**Reply by Oniara** [reply]
Hello there! Thanks for sharing this, I believe this will do for now. Tho it would be nice to have a direct plugin that would work with videojs or even better native support for such subtitles one day.

Anyways, thanks again, best wishes!
